src/Text/Pretty/Simple/Internal/Printer.hs

```diff
@@ -146,7 +146,7 @@ data OutputOptions = OutputOptions
   --
   -- You can see that all the escape characters get output literally, including
   -- newline.
-  } deriving (Eq, Generic, Show, Typeable)
+  } deriving (Generic, Typeable)
```

Comment:
Actually, introducing a concrete `Postproc` type would at least allow us to recover some form of `Show`

Reply:
Not having some way to `Show` an `OutputOptions` would be somewhat annoying for someone playing around with `pretty-simple` in the repl.

I'd be fine with hand-writing a `Show` instance for `OutputOptions` where we just have the `outputOptionsPostProcess` be shown as something like `outputOptionsPostProcess = <...>`.

Although this is a good reason for making `[Expr] -> [Expr]` be a newtype (because this would give us a derivable `Show` instance).